### DISCUSSION TITLE
What is the "tone" noise when altitude changes

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3950] [Q&A] [answered]
When I'm over the transition altitude and I forget to hit STD on the pressure knob, and then I press it and find my altitude needs adjusting, there is a single, long, tone that sounds until my altitude is corrected for STD pressure. Any idea how to manage it (turn it off)?

**Comment by Paytheon** [comment]
Press the button just above the master caution light. The tone will turn off.

**Reply by robinottawa** [reply]
Thanks.

**Comment by tracernz** [comment, marked as answer]
It’s called the C chord. Warns you when about to arrive at set altitude, and when deviating from set altitude.
